vowpalwabbit/multilabel_oaa.cc

```diff
@@ -50,12 +53,36 @@ void predict_or_learn(multi_oaa& o, VW::LEARNER::single_learner& base, example&
                         multilabels.label_v[multilabel_index], o.k - 1);
     }
   }
-  ec.pred.multilabels = preds;
-  ec.l.multilabels = multilabels;
+  if (o.probabilities)
+  {
+    assert(ec.pred.scalars.size() == o.k);
+    float sum = std::accumulate(ec.pred.scalars.begin(), ec.pred.scalars.end(), 0.f);
```

Comment:
On the next line (it won't let me comment there?) I don't think we should be normalize the probabilities.  In a multilabel context, the appropriate normalizer is the expected number of labels, not 1.